libs/coin-modules/coin-hedera/src/deviceTransactionConfig.test.ts

```diff
@@ -0,0 +1,310 @@
+import BigNumber from "bignumber.js";
+import type { Account } from "@ledgerhq/types-live";
+import { HEDERA_TRANSACTION_MODES } from "./constants";
+import getDeviceTransactionConfig from "./deviceTransactionConfig";
+import type { Transaction, TransactionStatus } from "./types";
+
+describe("getDeviceTransactionConfig", () => {
+  const mockAccount = {
+    id: "mock-account-id",
+    currency: { id: "hedera" },
+  } as Account;
+
+  const createMockStatus = (estimatedFees: BigNumber): TransactionStatus => ({
+    errors: {},
+    warnings: {},
+    estimatedFees,
+    amount: new BigNumber(0),
+    totalSpent: new BigNumber(0),
+  });
+
+  describe("staking transactions", () => {
+    it("should return correct fields for ClaimRewards transaction", async () => {
+      const transaction: Transaction = {
+        family: "hedera",
+        mode: HEDERA_TRANSACTION_MODES.ClaimRewards,
+        amount: new BigNumber(0),
+        recipient: "",
+        memo: "Claiming rewards",
+      } as Transaction;
+
+      const status = createMockStatus(new BigNumber(100000));
+
+      const fields = await getDeviceTransactionConfig({
+        account: mockAccount,
+        transaction,
+        status,
+      });
+
+      expect(fields).toHaveLength(2);
+      expect(fields).toMatchObject([
```

Comment:
```suggestion
      expect(fields).toEqual([
```
Tip for the whole file: no need to test the length of the array if you also test it is equal or even partially equal to another array of your expected length